static/css/docs.scss

```diff
@@ -11,20 +11,52 @@
 @include docs-wrapper;
 @include docs-toc;
 
-.documentation__nav h4 {
-  margin-top: 0.2em;
-  margin-bottom: 0.2em;
+.documentation__wrapper {
+  flex-direction: column;
+
+  @media only screen and (min-width : $breakpoint-medium) {
+    flex-direction: column;
+  }
 }
 
-.documentation__nav ul li:after {
-  height: 2.25em;
+.documentation__jump {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
```

Comment:
@anthonydillon is that our convention? I thought it was better to always use expanded format because people so often accidentally override paddings, margins and borders by using the shorthand.